packages/core/src/components/viewer-grid/viewer-grid.tsx

```diff
@@ -41,6 +41,9 @@ export class ViewerGrid {
   @Prop({ reflect: true })
   overlayOpen = false;
 
+  @Prop()
+  initialMainSize: MainSize | undefined;
+
   @State()
   mainSize: MainSize = "large";
```

Comment:
```suggestion
  mainSize: MainSize = this.initialMainSize ?? "large";
```

Zou dit werken en dan zonder jouw code in de `connectedCallback()` ?

Reply:
Aangepast